ApexDocContent/API.htm

```diff
@@ -12,7 +12,8 @@ <h2>Nonprofit Success Pack API (Application Programming Interface)</h2>
 	
 		<p>The classes involved with the public interface include:</p>
 		<ul>
-		    <li>BDI_DataImport_API</li>
+			<li>BDI_DataImport_API</li>
+			<li>CallableApi</li>
 			<li>ERR_Handler_API</li>
```

Comment:
@sam-knox - I simply added the new class to this HTML page - is there more info that we need to do - with regards to informing the users of the new API?

Reply:
FYI: We still keep these files up to date, but technically they're only used by the currently non-functional ApexDoc system. The hope is that one day ApexDocs will start generating docs for our app again.

Reply:
@patrick-yan-sf - That should be sufficient. As Mike says, ApexDoc is defunct so the repo itself is the best place for people to look.